src/screens/proposal/index.tsx

```diff
@@ -404,7 +404,7 @@ function Proposal() {
                         variant='v2_subheading'
                         fontWeight='500'
                         ml='auto'>
-                        {getRewardAmountMilestones(proposal?.token?.decimals!, proposal)}
+                        {parseInt(getRewardAmountMilestones(proposal?.token?.decimals!, proposal)).toLocaleString()}
```

Comment:
Didn't get why we're converting to int & then back to string?

Reply:
We want to show 16000 USD as 16,000 USD, i.e., with proper commas. `toLocaleString()` is being used for comma insertion. But the method `toLocaleString()` accepts number values and not strings, hence converting to Int first.